### PR TITLE
CB-129: Clarify setup instructions in the README.md

### PR DESCRIPTION
**Pull request by jomadoye**
# JIRA TICKET NAME:
CB-129: [Clarify setup instructions in the README.md](https://issues.openmrs.org/browse/CB-129)

## SUMMARY:
This PR clarifies the setup instructions in the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ npm install
 # Copy the webpack configuration from webpack.config.sample.js to webpack.config.js
 cp webpack.config.sample.js webpack.config.js
 
-# Locate appdata\owa folder and copy the path
+# Locate the 'appdata\owa' directory in the reference application folder and type the following command to get the path to the 'appdata\owa' directory and copy it to the clipboard
```
Comment (dkayiwa) [inline README.md]: Can you give an example of what this could be on your system?

Comment (jomadoye) [reply README.md]: @dkayiwa an example of the present working directory is /Users/andeladeveloper/Downloads/referenceapplication-standalone-2.6.0/appdata\owa

Comment (jomadoye) [reply README.md]: the `pbcopy` copies it to the clipboard

Comment (dkayiwa) [reply README.md]: I mean putting the example directly in the readme. That will give some one an idea in regards to where to look.

Comment (jomadoye) [reply README.md]: @dkayiwa, I have added it.

File: README.md [hunk 2]
```diff
@@ -31,9 +31,11 @@ npm install
 # Copy the webpack configuration from webpack.config.sample.js to webpack.config.js
 cp webpack.config.sample.js webpack.config.js
 
-# Locate appdata\owa folder and copy the path
+# Locate the 'appdata\owa' directory in the reference application folder and type the following command to get the path to the 'appdata\owa' directory and copy it to the clipboard.
```
Comment (dkayiwa) [inline README.md]: Does it have to be only the reference application?

File: README.md [hunk 3]
```diff
@@ -31,9 +31,11 @@ npm install
 # Copy the webpack configuration from webpack.config.sample.js to webpack.config.js
 cp webpack.config.sample.js webpack.config.js
 
-# Locate appdata\owa folder and copy the path
+# Locate the 'appdata\owa' directory and type the following command to get the path to the 'appdata\owa' directory and copy it to the clipboard.
 pwd | pbcopy
 
+An example of the path is: /Users/andeladeveloper/Downloads/referenceapplication-standalone-2.6.0/appdata\owa/
```
Comment (dkayiwa) [inline README.md]: For your example, did you intentionally mix the forward and backward slashes?

Comment (jomadoye) [reply README.md]: No @dkayiwa, the name of the folder is `appdata\owa`.

Comment (dkayiwa) [reply README.md]: With forward or backward slash?

Comment (jomadoye) [reply README.md]: with forward slash.

Comment (jomadoye) [reply README.md]: screen shot 2017-08-08 at 3 16 53 pm

Comment (jomadoye) [reply README.md]: @dkayiwa a screenshot is above

Comment (dkayiwa) [reply README.md]: a folder name with a backward slash is a bit confusing. Can you use the same slash as for the rest of the path?

Comment (jomadoye) [reply README.md]: Fixed, @dkayiwa

File: README.md [hunk 4]
```diff
@@ -31,9 +31,11 @@ npm install
 # Copy the webpack configuration from webpack.config.sample.js to webpack.config.js
 cp webpack.config.sample.js webpack.config.js
 
-# Locate appdata\owa folder and copy the path
+# Locate the 'appdata\owa' directory and type the following command to get the path to the 'appdata\owa' directory and copy it to the clipboard.
```
Comment (dkayiwa) [inline README.md]: The above still has a backward slash

Comment (jomadoye) [reply README.md]: Fixed @dkayiwa.

**Pull request merged**